seed/static/seed/js/controllers/organization_settings_controller.js

```diff
@@ -128,6 +126,12 @@ angular.module('SEED.controller.organization_settings', []).controller('organiza
       unit: null
     };
 
+    const property_ubid = $scope.property_columns.find((c) => c.column_name === 'ubid');
+    const taxlot_ubid = $scope.taxlot_columns.find((c) => c.column_name === 'ubid');
```

Comment:
Is all the column business just for this?

Reply:
Yeah, its to get matching_criteria column info. The original `property_column_names` is a list of just id, column_name, and display_name, instead of the whole column dict.